layerfetcher/layers.go

```diff
@@ -305,6 +305,28 @@ func (l *Logic) PollLayerContent(ctx context.Context, layerID types.LayerID) cha
 	return resChannel
 }
 
+// registerLayerHashes registers provided hashes with provided peer.
+func (l *Logic) registerLayerHashes(peer p2p.Peer, blocks *layerData) {
+	if blocks == nil {
```

Comment:
rename blocks to something else. it's from very old code.

Reply:
done